inst/www/shiny.fluent/shiny-fluent.js.LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+/*! *****************************************************************************
```

Comment:
hmm, is this in the right place? I mean, it looks as if this file is a license for the entire shiny.fluent, not only for the Fluent UI. It's named `shiny-fluent.js.LICENSE.txt `... Why do we need this?

Reply:
Webpack generates this file along `shiny-fluent.js`, as it has Fluent UI bundled in it.